admin_guide/assembly_restore-etcd-quorum.adoc

```diff
@@ -0,0 +1,66 @@
+[[assembly_restore-etcd-quorum]]
+= Restoring etcd quorum
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: restore-etcd-quorum
+
+toc::[]
+
+
+If you lose etcd quorum, you must take down your etcd cluster and form a new 
+one. You can use one healthy etcd node to form a new cluster, but you must
+remove all other healthy nodes.
+
+[NOTE]
+====
+During etcd quorum loss, applications that run on {product-title} are
+unaffected. However, the platform functionality is limited to read-only
+operations. You cannot take action such as scaling an application up or down, 
+changing deployments, or running or modifying builds.
+====
+
+To confirm the loss of etcd quorum, run the following command and confirm
+that the cluster is unhealthy:
+
+----
+# ETCDCTL_API=2 etcdctl  --cert-file=/etc/origin/master/master.etcd-client.crt  \
+          --key-file /etc/origin/master/master.etcd-client.key \
+          --ca-file /etc/origin/master/master.etcd-ca.crt \
+          --endpoints="https://*master-0.example.com*:2379,\
+          https://*master-1.example.com*:2379,\
+          https://*master-2.example.com*:2379"\
+          cluster-health
+
+member 165201190bf7f217 is unhealthy: got unhealthy result from https://master-0.example.com:2379
+member b50b8a0acab2fa71 is unreachable: [https://master-1.example.com:2379] are all unreachable
+member d40307cbca7bc2df is unreachable: [https://master-2.example.com:2379] are all unreachable
+cluster is unhealthy
+----
+
+Note the member IDs and host names of the hosts. You use one of the nodes that
+can be reached to form a new cluster.
+
```

Comment:
Are all of these sections required? I assume so, but do we want to be more clear? 
_If you lose etcd quorum, perform all of the following tasks, based on your etcd version_

Reply:
@mburke5678, this is an interesting question - I'd normally say no, but I'm used to all topics reflecting standalone user flows. Only the last step is optional.

If I use this paragraph at line 16, do you think it still needs a clearer note at 46? 
```
If you lose etcd quorum, you must back up etcd, take down your etcd cluster,
and form a new one. You can use one healthy etcd node to form a new cluster, 
but you must remove all other healthy nodes.
```

Reply:
@kalexand-rh LGTM!